### ISSUE TITLE
Open ALPR Build Error

### ISSUE THREAD

**Issue by sohanseervi** [twelve17/openalpr-ios#8]
Hi,
I goes through all steps mentioned in read me file.
I am getting one error when building the xcode project:
clang: error: no such file or directory: '../openalpr-ios/openalpr-xcode/openalpr/openalpr/segmentation/segment.cpp'
clang: error: no input files
Command /Applications/Xcode_6_2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang failed with exit code 1

Can you let me know how to resolve this error?

**Comment by twelve17** [comment]
For v2 of this project, I have replaced the old instructions and bash scripts with a Ruby script to build the OpenALPR and dependency Frameworks, as well as the OpenALPR Xcode project.  A sample app is included.  These changes have been pushed to the master branch, and the old version has been moved to branch `1_0`.  Feel free to check out this new script and see if it works for you.  I will no longer be working on or supporting the `1_0` codebase.
